### DISCUSSION TITLE
Builds Manager Module Profile

### DISCUSSION THREAD

**Discussion by KenediaDev** [KenediaDev/Kenedia.BlishHUD#6] [Blish Modules]
![Preview](https://github.com/user-attachments/assets/ab544f05-72e1-4b02-8ecb-d7b5bcce70a4)

Sadly ArenaNet decided to not give us as amazing build templates as we had in the orginal Guild Wars game. Since we don't want to play with a Browser Tab open the whole time and don't want to have only a few builds per alt character, this module was created.

**This is the module which will make your life a lot easier if you like to play a lot of different builds!**

# FAQ | Manual

### **Q: I can't copy my gear template to the game.**

A: Unfortunately, the game client does not expose the actual gear template code, so it is not possible for the module to apply the template directly in-game.  
Because of this restriction, the template system is strictly an internal feature. You can still use it to share your template quickly with others, but the **Gear Templates** tab itself is meant purely for your own reference — a place to store and organize information about your builds and equipment.

## Quick Filter Panel
![Quick Filter Panel](https://github.com/user-attachments/assets/e3a88f03-6515-4bc7-a18f-89ae68406d10)

## About
![AboutTab](https://github.com/user-attachments/assets/80ad97e9-a088-4205-a2c7-f508fdc9b78d)

## Build
![BuildTab](https://github.com/user-attachments/assets/5c71a778-0757-45ba-9fc6-59a98131ee9d)

## Equipment
![EquipmentTab](https://github.com/user-attachments/assets/b02d34d7-a81e-4bd4-843c-7be539709274)

## Tags and Groups
![Tags and Groups](https://github.com/user-attachments/assets/c24e8b95-98db-48e0-92ca-cf4073a8c38c)

## Settings
![Settings](https://github.com/user-attachments/assets/96e336d9-c918-4d75-8a11-313e8e35202d)

**Comment by Aze0s** [comment]
The Build Module is actually down since last update :(
